### PR TITLE
Bugfix/fix cpu only tests

### PR DESCRIPTION
**Pull request by badisa**

### CODE REVIEW

File: timemachine/parallel/utils.py [hunk 1]
```diff
@@ -8,7 +8,10 @@
 
 def get_gpu_count() -> int:
     # Expected to return a line delimited summary of each GPU
-    output = check_output(["nvidia-smi", "-L"])
+    try:
+        output = check_output(["nvidia-smi", "-L"])
+    except FileNotFoundError:
```
Comment (badisa) [inline timemachine/parallel/utils.py:13]: Perhaps a log statement after this? Didn't want it to be noisy, but can see an argument for one.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -52,6 +52,7 @@ nogpu-tests:
   needs: ["lint"]
   tags:
     - timemachine
+    - cpu
```
Comment (badisa) [inline .gitlab-ci.yml:55]: Truly run on a machine without a GPU, and without any of cuda drivers/nvidia-smi/etc

**Review by proteneer — APPROVED** [review]
lgtm!

File: timemachine/parallel/utils.py (hunk 1 above)
Comment (badisa) [reply timemachine/parallel/utils.py:13]: Driver errors will still raise an exception. This only returns 0 if there is no binary for `nvidia-smi`

**Pull request merged**